modules/bitgo/test/v2/unit/keychains.ts

```diff
@@ -76,6 +76,7 @@ describe('V2 Keychains', function () {
         n.asset !== UnderlyingAsset.OPETH && // TODO(WIN-692): remove this once coin-specific module for opeth is added
         n.asset !== UnderlyingAsset.ZKETH && // TODO(WIN-1427): remove this once coin-specific module for zketh is added
         n.asset !== UnderlyingAsset.OAS && // TODO(WIN-3696): remove this once coin-specific module for oas is added
+        n.asset !== UnderlyingAsset.COREDAO && // TODO(WIN-3696): remove this once coin-specific module for oas is added
```

Comment:
Please update the copied comment....

Reply:
Have update the comments, The jira ticket will remain the same since oas and coredao have same ticket for SDK implementation